### PR TITLE
Feature/juan pulido/jumping from a wall

### PR DESCRIPTION
**Pull request by JPulido01**
It's complete

### CODE REVIEW

File: Assets/Scripts/Movement.cs
Comment (Loko24) [inline Assets/Scripts/Movement.cs:1]: Private variables should start with _ (_speed)

**Review by Loko24 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Movement.cs
Comment (JPulido01) [reply Assets/Scripts/Movement.cs:1]: Ready

**Review by Nik-Stevens — APPROVED** [review]
Approved

**Review by Nik-Stevens — APPROVED** [review]
Approved

**Review by daniel374 — APPROVED** [review]

**Review by Loko24 — APPROVED** [review]

**Pull request merged**